scripts/install_third_party_dependencies.sh

```diff
@@ -1,49 +1,20 @@
 #!/bin/bash
-CONDA_INSTALL_URL=${CONDA_INSTALL_URL:-"https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh"}
-
-source scripts/vars.sh
-
-# Install Miniconda locally
-rm -rf lib/conda
-rm -f /tmp/Miniconda3-latest-Linux-x86_64.sh
-wget -P /tmp \
-    "${CONDA_INSTALL_URL}" \
-    && bash /tmp/Miniconda3-latest-Linux-x86_64.sh -b -p lib/conda \
-    && rm /tmp/Miniconda3-latest-Linux-x86_64.sh
-
-# Grab conda-only packages
-export PATH=lib/conda/bin:$PATH
-lib/conda/bin/python3 -m pip install nvidia-pyindex
-conda env create --name=${ENV_NAME} -f environment.yml
-source scripts/activate_conda_env.sh
-
-echo "Attempting to install FlashAttention"
-git clone https://github.com/HazyResearch/flash-attention
-CUR_DIR=$PWD
-cd flash-attention
-git checkout 5b838a8bef
-python3 setup.py install
-cd $CUR_DIR
-
-# Install DeepMind's OpenMM patch
-OPENFOLD_DIR=$PWD
-pushd lib/conda/envs/$ENV_NAME/lib/python3.9/site-packages/ \
-    && patch -p0 < $OPENFOLD_DIR/lib/openmm.patch \
-    && popd
 
 # Download folding resources
-wget --no-check-certificate -P openfold/resources \
+wget -N --no-check-certificate -P openfold/resources \
     https://git.scicore.unibas.ch/schwede/openstructure/-/raw/7102c63615b64735c4941278d92b554ec94415f8/modules/mol/alg/src/stereo_chemical_props.txt
 
 # Certain tests need access to this file
 mkdir -p tests/test_data/alphafold/common
 ln -rs openfold/resources/stereo_chemical_props.txt tests/test_data/alphafold/common
 
-echo "Downloading OpenFold parameters..."
-bash scripts/download_openfold_params.sh openfold/resources
-
-echo "Downloading AlphaFold parameters..."
-bash scripts/download_alphafold_params.sh openfold/resources
-
 # Decompress test data
 gunzip -c tests/test_data/sample_feats.pickle.gz > tests/test_data/sample_feats.pickle
+
+python setup.py install
+
+export LD_LIBRARY_PATH=$CONDA_PREFIX/lib:$LD_LIBRARY_PATH
+
+# The following will prepend conda library environment to $LD_LIBRARY_PATH
+# upon conda library activation 
+# conda env config vars set LD_LIBRARY_PATH=$CONDA_PREFIX/lib:$LD_LIBRARY_PATH
```

Comment:
Not sure about this, I guess $LD_LIBRARY_PATH could be different on diff nodes, this will override the one set on that node with whatever value it had when you first save the env var.

Reply:
Yeah, I think you're right, and this is more trouble than it's worth. The first time you deactivate from the environment, it seems to unset the LD_LIBRARY_PATH entirely which can be a bit concerning if you're not expecting that.